presto-main/src/test/java/com/facebook/presto/sql/planner/assertions/FunctionCallProvider.java

```diff
@@ -35,35 +40,47 @@
     private final Optional<WindowFrame> frame;
     private final boolean distinct;
     private final List<PlanTestSymbol> args;
+    private final List<PlanMatchPattern.Ordering> orderBy;
 
-    private FunctionCallProvider(boolean isWindowFunction, QualifiedName name, Optional<WindowFrame> frame, boolean distinct, List<PlanTestSymbol> args)
+    private FunctionCallProvider(boolean isWindowFunction, QualifiedName name, Optional<WindowFrame> frame, boolean distinct, List<PlanTestSymbol> args, List<PlanMatchPattern.Ordering> orderBy)
     {
         this.isWindowFunction = isWindowFunction;
         this.name = requireNonNull(name, "name is null");
         this.frame = requireNonNull(frame, "frame is null");
         this.distinct = distinct;
         this.args = requireNonNull(args, "args is null");
+        this.orderBy = requireNonNull(orderBy, "orderBy is null");
     }
 
     FunctionCallProvider(QualifiedName name, Optional<WindowFrame> frame, boolean distinct, List<PlanTestSymbol> args)
     {
-        this(true, name, frame, distinct, args);
+        this(true, name, frame, distinct, args, ImmutableList.of());
     }
 
     FunctionCallProvider(QualifiedName name, boolean distinct, List<PlanTestSymbol> args)
     {
-        this(false, name, Optional.empty(), distinct, args);
+        this(false, name, Optional.empty(), distinct, args, ImmutableList.of());
+    }
+
+    FunctionCallProvider(QualifiedName name, List<PlanTestSymbol> args, List<PlanMatchPattern.Ordering> orderBy)
+    {
+        this(false, name, Optional.empty(), false, args, orderBy);
     }
 
     FunctionCallProvider(QualifiedName name, List<PlanTestSymbol> args)
     {
-        this(false, name, Optional.empty(), false, args);
+        this(false, name, Optional.empty(), false, args, ImmutableList.of());
     }
 
     @Override
     public String toString()
     {
-        return format("%s%s (%s) %s", distinct ? "DISTINCT" : "", name, Joiner.on(", ").join(args), frame.isPresent() ? frame.get().toString() : "");
```

Comment:
It is easier to review code when formatting changes are separated from actual logical changes

Reply:
Agreed. Some cases are tricky, though, such as when you have to change the format for the new code to be readable, but the old code doesn't need it (e.g., when adding a parameter to a method that makes the declaration too long).